specs/_features/eip7805/validator.md

```diff
@@ -0,0 +1,147 @@
+
+## Prerequisites
+
+This document is an extension of the [Electra -- Honest Validator](../../electra/validator.md) guide.
+All behaviors and definitions defined in this document, and documents it extends, carry over unless explicitly noted or overridden.
+
+All terminology, constants, functions, and protocol mechanics defined in the updated Beacon Chain doc of [EIP-7805](./beacon-chain.md) are requisite for this document and used throughout.
+Please see related Beacon Chain doc before continuing and use them as a reference throughout.
+
+### Time parameters
+
+| Name | Value | Unit | Duration |
+| - | - | :-: | :-: |
+| `PROPOSER_INCLUSION_LIST_CUT_OFF` | `uint64(11)` | seconds | 11 seconds |
+
+## Protocol
+
+### `ExecutionEngine`
+
+*Note*: `engine_getInclusionListV1` and `engine_updateBlockWithInclusionListV1` functions are added to the `ExecutionEngine` protocol for use as a validator.
+
+The body of these function is implementation dependent. The Engine API may be used to implement it with an external execution engine. 
+
+## New inclusion list committee assignment
+
+A validator may be a member of the new Inclusion List Committee (ILC) for a given slot. To check for ILC assignments the validator uses the helper `get_inclusion_committee_assignment(state, epoch, validator_index)` where `epoch <= next_epoch`.
+
+Inclusion list committee selection is only stable within the context of the current and next epoch.
+
+```python
+def get_inclusion_committee_assignment(
+        state: BeaconState,
+        epoch: Epoch,
+        validator_index: ValidatorIndex) -> Optional[Slot]:
+    """
+    Returns the slot during the requested epoch in which the validator with index ``validator_index``
+    is a member of the ILC. Returns None if no assignment is found. 
+    """
+    next_epoch = Epoch(get_current_epoch(state) + 1)
+    assert epoch <= next_epoch
+
+    start_slot = compute_start_slot_at_epoch(epoch)
+    for slot in range(start_slot, start_slot + SLOTS_PER_EPOCH):
+        if validator_index in get_inclusion_list_committee(state, Slot(slot)):
+            return Slot(slot)
+    return None
+```
+
+### Lookahead
+
+`get_inclusion_committee_assignment` should be called at the start of each epoch to get the assignment for the next epoch (`current_epoch + 1`). A validator should plan for future assignments by noting their assigned ILC slot. 
+
+## New proposer duty
+
+### Block proposal
+
+Proposers are still expected to propose `SignedBeaconBlock` at the beginning of any slot during which `is_proposer(state, validator_index)` returns true. The mechanism to prepare this beacon block and related sidecars differs from previous forks as follows:
+
+#### Update execution client with inclusion lists
+
+The proposer should call `engine_updateInclusionListV1` at `PROPOSER_INCLUSION_LIST_CUT_OFF` into the slot with the list of the inclusion lists that gathered since `inclusion_list_CUT_OFF`.
```

Comment:
`inclusion_list_CUT_OFF` is not defined anywhere, is this supposed to be the cutoff at 9 seconds?

Reply:
Fixed. I believe it's supposed to be `PROPOSER_INCLUSION_LIST_CUT_OFF`.